simulator/src/main/java/com/hedera/block/simulator/BlockStreamSimulatorApp.java

```diff
@@ -31,16 +36,11 @@
 /** BlockStream Simulator App */
 public class BlockStreamSimulatorApp {
 
-    private static final System.Logger LOGGER =
-            System.getLogger(BlockStreamSimulatorApp.class.getName());
-
+    private final System.Logger LOGGER = System.getLogger(getClass().getName());
     private final BlockStreamManager blockStreamManager;
     private final PublishStreamGrpcClient publishStreamGrpcClient;
     private final BlockStreamConfig blockStreamConfig;
```

Comment:
I believe this config is currently only used within the constructor, maybe you can consider to have it only there and not as a field? Not a blocker, up to you